Signal/src/ViewControllers/HomeView/HomeViewController.m

```diff
@@ -308,25 +303,10 @@ - (void)loadView
 
 - (void)updateReminderViews
 {
-    BOOL shouldHideArchiveReminderView = self.homeViewMode != HomeViewMode_Archive;
```

Comment:
Since we're using a stack view, we can show/hide the subviews rather than track a separate constraint. I think this is simpler.

Reply:
Definitely simpler. Sorry, I didn't realize `UIStackView` would ignore hidden views.